src/brasil/gov/tiles/tiles/templates/carousel.pt

```diff
@@ -0,0 +1,57 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="brasil.gov.tiles">
+  <body tal:define="is_empty view/is_empty">
+    <p tal:condition="python: is_empty and view.is_compose_mode()" i18n:translate="">
+      Please add up to <span i18n:name="limit_objects" tal:replace="view/limit"/> objects to the tile.
+    </p>
+
+    <div class="brasil-carousel-tile tile-content"
+         tal:condition="not:is_empty"
+         tal:attributes="id string:carousel-${view/id}">
+      <div class="tile-header" tal:condition="view/tile_title">
+        <h2 tal:content="view/tile_title" />
+        <p tal:content="view/tile_description" />
+      </div>
+      <div class="switch-carousel" tal:attributes="data-text view/switch_text"></div>
+      <div class="swiper-container carousel-thumbs">
+        <div class="swiper-wrapper">
+          <tal:pages repeat="page view/results">
+            <div class="swiper-slide">
+              <tal:items repeat="obj page">
+                <div class="carousel-item"
+                     tal:attributes="data-tile-id view/id;
+                                     data-content-type obj/portal_type;
+                                     data-content-uuid python:view.get_uuid(obj)">
+                  <tal:image define="thumbnail python:view.thumbnail(obj);"
+                             condition="thumbnail">
+                    <a tal:attributes="href python:view.get_url(obj)">
+                      <img tal:attributes="src thumbnail/url|string:${item/absolute_url}/download;
+                                           width thumbnail/width;
+                                           height thumbnail/height;
+                                           class view/get_image_position;
+                                           alt python:view.get_alt(obj);" />
```

Comment:
in the future we'll need to review this attribute as the alternate text has to be entered explicitly.

Reply:
ok